artview/components/plot.py

```diff
@@ -112,23 +103,26 @@ def __init__(self, Vradar, Vfield, Vtilt, Vlims=None, \
         self.cm_names = [m for m in cm.datad if not m.endswith("_r")]
         self.cm_names.sort()
 
-        # Create a figure for output
-        self._set_fig_ax(nrows=1, ncols=1)
-
         # Create tool dictionary
         self.tools = {}
-        self.tools['zoompan'] = None
+
+        # Check the file type and initialize limts
+        # Create a figure for output
+        self._check_file_type()
```

Comment:
Ok, so there is a conflict between initialization and execution, this indicates we are doing something wrong.
From the start: when should the limits be reset?